Update xcodeproj

pietbrauer opened the pull request.

pietbrauer commented:
Not sure why `ROOT/bin/` is prefixed in all `execution_ouput.txt` files?

mrackwitz commented on Gemfile.lock:
```diff
@@ -5,30 +5,30 @@ PATH
     claide (0.9.1)
     clamp (0.6.5)
-    clintegracon (0.5.2)
+    clintegracon (0.8.1)
```
@pietbrauer: This comes likely through the version bump here. I'll take care of that.

pietbrauer commented:
@mrackwitz I reverted the update to `clintegracon` for now. I also updated the `Example.xcodeproj` `yml`-presentations, it seems to me as the `yml` files are generated using `xcodeproj` and these are valid changes.

mrackwitz commented:
Whoops! Totally forgot about your PR sitting here. Sorry Piet!

>I also updated the `Example.xcodeproj` `yml`-presentations, it seems to me as the `yml` files are generated using `xcodeproj` and these are valid changes.

Not exactly. The YAML representations are [generated by the test harness](https://github.com/mrackwitz/xcres/blob/master/spec/spec_helper/xcodeproj_project_yaml.rb), but they use `xcodeproj`'s hash representations. The serialization, which has changed here, depends usually on the libyaml version in the environment. Not sure exactly how that has changed. But the changes seem valid. :+1:
